prisma/schema.prisma

```diff
@@ -593,6 +599,33 @@ model AuthenticationAttempt {
   @@map("LoginAttempts")
 }
 
+model Passkey {
+  id            String   @id @default(uuid())
+  credentialId  String   @db.VarChar(255)
+  publicKey     String   @db.VarChar(1024)
+  signCount     Int      @default(0)
+  label         String   @db.VarChar(255)
+  createdAt     DateTime @default(now())
+  lastUsedAt    DateTime @default(now())
+
+  UserProfile   UserProfile @relation(fields: [userId], references: [supId], onDelete: Cascade)
+  userId        String      @db.Uuid
+
+  @@unique([userId, credentialId], name: "userPasskey")
+  @@map("Passkeys")
+}
+
+model PasskeyChallenge {
+  id        String   @id @default(uuid())
+  userId    String   @db.VarChar(255)
```

Comment:
Any reason to use `String` here instead of doing the same as with the `Passkey` model above?

```
  UserProfile   UserProfile @relation(fields: [userId], references: [supId], onDelete: Cascade)
  userId        String      @db.Uuid
```

This also avoids leaving orphan `PasskeyChallenge`s behind if a user is deleted.